src/_includes/migration.njk

```diff
@@ -85,21 +94,19 @@
     function dropHandler(event) {
 
         // Prevent default behavior (Prevent file from being opened)
-        ev.preventDefault()
+        event.preventDefault()
 
-        if (ev.dataTransfer.items) {
+        if (event.dataTransfer.items) {
             // Use DataTransferItemList interface to access the file(s)
-            [...ev.dataTransfer.items].forEach((item, i) => {
+            [...event.dataTransfer.items].forEach((item, i) => {
+                console.log(item)
                 // If dropped items aren't files, reject them
                 if (item.kind === "file") {
-                    d1Flow = item.getAsFile();
+                    const file = item.getAsFile();
+                    console.log(`… file[${i}].name = ${file.name}`)
```

Comment:
?

Reply:
Console log shouldn't be there. I'm now back to mobile, but if you can request line change, I can approve it

Reply:
```suggestion
```